app/globals.css

```
@@ -2,37 +2,80 @@
+@font-face {
+  font-family: "Material Symbols Outlined";
+  src: url("/fonts/MaterialSymbolsOutlined-Regular.woff2");
+}
+
+@font-face {
+  font-family: "Material Symbols Outlined Filled";
+  src: url("/fonts/MaterialSymbolsOutlined_Filled-Regular.woff2");
+}
+
+.material-symbols-outlined {
+  font-family: 'Material Symbols Outlined';
```

Biome (1.9.4)

[error] Generic font family missing.

Consider adding a generic font family as a fallback.
For examples and more information, see the MDN Web Docs


(lint/a11y/useGenericFontNames)